charts/airflow/values.yaml

```diff
@@ -559,6 +559,12 @@ scheduler:
     ##
     enabled: false
 
+    ## the `apiVersion` to use for PodDisruptionBudget resources
+    ## - for Kubernetes 1.21 and later: "policy/v1"
+    ## - for Kubernetes 1.20 and before: "policy/v1beta1"
+    ##
+    apiVersion: policy/v1beta1
```

Comment:
I think we should default to `policy/v1` (and also for the others), because it [was added in 1.21](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125), and the earliest currently supported version is already 1.23.